### DISCUSSION TITLE
RISC-V Support

### DISCUSSION THREAD

**Discussion by asifaligorikhan** [taosdata/TDengine#23016] [Q&A]
Is TDengine supports for riscv64 architechture ?
Can I port TDengine for RISC-V architecture ?

**Comment by ccc0** [comment]
Hello， port TDengine for RISC-V architecture，What is the result？

**Reply by aloscha1** [reply]
It shouldn't work because it TDengine doesn't support RISC-V but you could port it yourself with the right resources
